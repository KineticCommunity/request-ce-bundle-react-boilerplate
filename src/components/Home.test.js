import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { Home } from './Home';

describe('Home component', () => {
  test('verify snapshot', () => {
    const component = shallow(<Home />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
