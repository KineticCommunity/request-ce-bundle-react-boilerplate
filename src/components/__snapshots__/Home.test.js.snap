// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component verify snapshot 1`] = `
<div>
  Hello World!
</div>
`;
